### PR TITLE
Fix border-color only accepting named colors

### PR DESCRIPTION
**Pull request by kristerkari**
Using rgb/rgba with `border-color` is throwing an error.

ping @jacobp100

### CODE REVIEW

File: src/transforms/index.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ const background = tokenStream => ({
   backgroundColor: tokenStream.expect(COLOR),
 })
 const borderColor = directionFactory({
-  types: [WORD],
+  types: [WORD, COLOR],
```
Comment (jacobp100) [inline src/transforms/index.js]: Does this work instead?

```suggestion
  types: [COLOR],
```

**Comment by jacobp100** [comment]
Good catch!

File: src/transforms/index.js (hunk 1 above)
Comment (kristerkari) [reply src/transforms/index.js]: Yes, it does!

**Comment by jacobp100** [comment]
Any idea why the build's failing?

**Comment by kristerkari** [comment]
> Any idea why the build's failing?

Looks like some npm registry error

**Comment by kristerkari** [comment]
@jacobp100 I don't know what to do with the broken CI, I tried restarting the build a couple of times, but it always fails with the same error of something being wrong in the tar header of the `camelize` package. Seems quite weird.

**Comment by jacobp100** [comment]
I suspect we just wanna update the packages - we should do that before we ship in case this starts affecting other people

**Pull request merged**